stories/molecules/Dropdown.stories.tsx

```diff
@@ -2,138 +2,91 @@ import * as React from 'react';
-import { HiDotsVertical } from 'react-icons/hi';
 
-const styles = {
-    content: {
-        display: 'flex',
-        justifyContent: 'center',
-        padding: '0 40px',
-    },
-    customDiv: {
-        alignItems: 'center',
-        background: '#ffff003d',
-        display: 'flex',
-        height: '40px',
-        justifyContent: 'center',
-    },
-    customListItem: {
-        alignItems: 'center',
-        display: 'flex',
-    },
-    divider: {
-        borderBottom: '1px solid #80808045',
-    },
-    icon: {
-        marginRight: '10px',
-    },
-};
-
-const meta: Meta<typeof Dropdown> = {
+const meta: Meta<typeof DropdownContent> = {
     title: 'Molecules/Dropdown',
-    component: Dropdown,
+    component: DropdownContent,
 };
 
 export default meta;
 
-type Story = StoryObj<typeof Dropdown>;
+type Story = StoryObj<typeof DropdownContent>;
 
 export const _Dropdown: Story = {
-    name: 'Dropdown',
+    name: 'Actions Dropdown',
     args: {},
-    render: (args) => {
-        const onDropdownStateChange = (object) => {
-            console.log(
-                'onDropdownStateChange was called with the following:',
-                JSON.stringify(object)
-            );
-        };
-
-        const customButtonsDemo = [
-            <Button context="primary">Click me!</Button>,
-            <Button>
-                <HiDotsVertical />
-            </Button>,
-            <Button variant="ghost">Select any</Button>,
-        ];
-
-        return (
-            <div style={styles.content}>
-                <Dropdown<string>
-                    {...args}
-                    button={customButtonsDemo[0]}
-                    additionalSelectProps={{
-                        onStateChange: onDropdownStateChange,
-                    }}
-                >
-                    <ListItem key="disabled-key" disabled style={styles.divider}>
-                        Disabled ListItem
-                    </ListItem>
-                    <Separator title="Section title" />
-                    <ListItem key="first-key" value="first-value">
-                        ListItem with value
-                    </ListItem>
-                    <Separator />
-                    {['ListItem with value 1', 'ListItem with value 2'].map((value) => (
-                        <ListItem key={value} value={value}>
-                            {value}
-                        </ListItem>
-                    ))}
-
-                    <ListItem key="second-key" value="second-value">
-                        <div style={styles.customListItem}>
-                            <IconTextkernel style={styles.icon} />
-                            <strong>Custom ListItem with value</strong>
-                        </div>
-                    </ListItem>
-                    <div style={styles.customDiv}>Just custom div element</div>
-                </Dropdown>
-            </div>
-        );
-    },
+    render: () => <p>To be implemented</p>,
```

Comment:
Maybe we should do better then this here. Or if not relevant, let's make a followup ticket for it.